modules/acs-quick-install-secured-cluster-using-helm.adoc

```diff
@@ -22,6 +22,7 @@ To install Collector on systems that have Unified Extensible Firmware Interface
 ====
 
 .Prerequisites
+* You must have generated {product-title-short} init bundle for your cluster.
```

Comment:
Just a note that "have generated" is passive voice. I'm not sure that active voice is necessary here, but thought it was worth mentioning. :slightly_smiling_face: